src/js/themes/hpe.js

```diff
@@ -464,9 +465,16 @@ export const hpe = deepFreeze({
         }
       `,
     },
-    extend: ({ disabled, theme }) => `
+    // focusIndicator is "false" when CheckBox is in a FormField.
+    // So, only apply hover background & horizontal pad when in FormField.
+
+    // NOTE: This won't work if teams have wrapped the Grommet input
+    // and named it "EZCheckBox" (for example), since FormField specifically
+    // looks for "CheckBox".
+    extend: ({ disabled, focusIndicator, theme }) => `
       ${
         !disabled &&
+        !focusIndicator &&
```

Comment:
We should get feedback on what the hover treatment should be if it's not in a FormField.

Also, this is a good "quick solution" to try to provide a solution for office hours, but we can discuss with @ericsoderberghp if we want  to think of something more elegant.

Reply:
agree I was thinking we can change to have a specific `pad` for when in formfield vs when its not but have that in theme but open to other options